manifests/init.pp

```diff
@@ -203,6 +203,8 @@
 #
 # $proxy_password::             Proxy password for authentication
 #
+# $yum_max_speed::              Maximum bandwidth used per download in bytes/sec
```

Comment:
this is also per worker right?  we may want to make that clear in the description